ArcMovies/Modules/MovieDetails/View/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  MovieDetailsViewController.swift
+//  ArcMovies
+//
+//  Created by Yuri Saboia Felix Frota on 21/01/18.
+//  Copyright © 2018 YuriFrota. All rights reserved.
+//
+
+import UIKit
+
+enum MovieDetails {
+    case header
+    case overview
+    case basicInfo
+}
+
+class MovieDetailsViewController: UIViewController, DetailsInterface {
+    var presenter: DetailsPresenter!
+    var table: UITableView!
+    var detailedMovie: Movie? {
+        didSet {
+            self.navigationItem.title = detailedMovie?.title ?? "Movie Details"
+            table.reloadData()
+        }
+    }
+    var details: [MovieDetails] = [.header,  .basicInfo, .overview]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        presenter.updateView()
+    }
+    
+    fileprivate func setupView() {
+        table = TableCreator.createTable(onView: self.view)
```

Comment:
For the screens implemented in this features, I didn't need a storyboard, it would only be three with a collection or table centralised, it wouldn't give much visual feedback, maybe for mor complex screens that needed more auto layout rules, I could've used a storyboard, but for this features was much simpler for me to use only xibs, that are extremely reusable, and some few lines of extra code for view and layout setups.
Storyboard can also prove to be a pain in the ass when working with bigger teams and merging branches, I had a couple of headaches in some projects doing merges because of storyboard conflicts, my approach now it's: only use it when needed.